modules/core/src/test/java/org/apache/fluo/core/impl/TransactionImplTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.fluo.core.impl;
+
+import org.apache.fluo.api.client.FluoClient;
+import org.apache.fluo.api.client.FluoFactory;
+import org.apache.fluo.api.client.Transaction;
+import org.apache.fluo.api.config.FluoConfiguration;
+import org.apache.fluo.api.data.Column;
+import org.apache.fluo.core.client.FluoClientImpl;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TransactionImplTest {
```

Comment:
This test should be in a different place. Fluo has Integration test (called ITs) that run against a real Fluo instance (backed by real Accumulo and Zookeeper instances).  These test are at `fluo/modules/integration/src/test/java/org/apache/fluo/integration`.  I would recommend putting your test there under client or impl and extending ITBaseImpl.  After that each test should have a FluoClient called client available.  So should be able to do the following.

```java
@Test
 public void testBasic() {
     try (Transaction tx = client.newTransaction()) {
          tx.set("row1", new Column("col1"), "val1");
          tx.set("row2", new Column("col2"), "val2");
￼          tx.set("row3", new Column("col3"), "val3");
￼ 
          tx.commit();
     }

     try (Transaction tx = client.newTransaction()) {
         //do async gets....
     }
     
}
```